agent/app/agent_capability.go

```diff
@@ -52,6 +52,7 @@ const (
 //    com.amazonaws.ecs.capability.task-iam-role-network-host
 //    ecs.capability.task-eni
 //    ecs.capability.task-eni-block-instance-metadata
+//    ecs.capability.task-execution-role
```

Comment:
I don't see this set anywhere.

Also, it should be something like `ecs.capability.task-execution-role.ecr-auth` to indicate that the only thing the task execution role can be used for is ECR authentication.

Reply:
fixed, but I need to confirm with the service team about the name.